flow-decomposition/src/main/java/com/powsybl/flow_decomposition/LossesCompensator.java

```diff
@@ -41,28 +43,84 @@ private boolean hasP0s(Branch<?> branch) {
     }
 
     void run(Network network) {
+        addNullLoadsOnBuses(network);
```

Comment:
avoid the use of "null" word which has a specific meaning in Java ... perhaps ZeroMwLoad ?

Reply:
done